collect_app/src/test/java/org/odk/collect/android/activities/MainActivityTest.java

```diff
@@ -82,54 +77,4 @@ public void manageFilesButtonTest() throws Exception {
         assertEquals(DeleteSavedFormActivity.class.getName(),
                 shadowIntent.getIntentClass().getName());
     }
-
-    @Test
-    public void shouldButtonsBeResistantToMulticlicking() {
```

Comment:
This test started failing when I disabled the multiclick guard in tests (so that other tests wouldn't fail). We've generally not been testing the multiclick guards (which I think makes sense) so personally I don't think we need this. Happy to chat about it though!